packages/service-clients/end-to-end-tests/azure-client/src/test/AzureClientFactory.ts

```diff
@@ -72,3 +73,80 @@ export function createAzureClient(
 		configProvider,
 	});
 }
+
+/**
+ * This function is used to create a container using any summary payload.
+ * It is primary intended as a workaround to using the AzureClient, and
+ * is only being used at the moment for ephemeral container E2E tests
+ * since AzureClient does not currently support ephemeral containers.
+ *
+ * Usage of this function for anything other than ephemeral E2E tests is
+ * not recommended.
+ *
+ * @param requestPayload - The summary payload used to create the container,
+ * currently these are mainly fetched from ephemeralSummaryTrees.ts
+ * @param userID - ID for the user creating the container
+ * @param userName - Name for the user creating the container
+ * @returns - An AxiosResponse containing the container ID(response.data.id)
+ */
+export async function createContainerFromPayload(
+	requestPayload: object,
+	userID?: string,
+	userName?: string,
+): Promise<AxiosResponse> {
+	const useAzure = process.env.FLUID_CLIENT === "azure";
+	const tenantId = useAzure
+		? (process.env.azure__fluid__relay__service__tenantId as string)
+		: "frs-client-tenant";
+	const user = {
+		id: userID ?? uuid(),
+		name: userName ?? uuid(),
+	};
+	const endPoint = useAzure
+		? (process.env.azure__fluid__relay__service__endpoint as string)
+		: "http://localhost:7071";
+	if (useAzure && endPoint === undefined) {
+		throw new Error("Azure FRS endpoint is missing");
+	}
+
+	const tokenProvider = useAzure
+		? createAzureTokenProvider(userID ?? "foo", userName ?? "bar")
+		: new InsecureTokenProvider("fooBar", user);
+	const ordererToken = await tokenProvider.fetchOrdererToken(tenantId, undefined, false);
+
+	const headers = {
+		"Authorization": `Basic ${ordererToken.jwt}`,
+		"Content-Type": "application/json",
+	};
+
+	const url = `/documents/${tenantId}`;
+
+	const options: AxiosRequestConfig = {
+		baseURL: endPoint,
+		data: requestPayload,
+		headers,
+		maxBodyLength: 1048576000,
+		maxContentLength: 1048576000,
+		method: "POST",
+		url,
+	};
+
+	try {
+		const response: AxiosResponse = await Axios(options);
+
+		if (response.status === 201) {
+			console.log("Container created successfully");
+		} else {
+			throw new Error(`Error creating container. Status code: ${response.status}`);
+		}
+
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+		if (response?.data?.id === undefined) {
```

Comment:
Tinylicious returns `id` as just a string, not an object with `id` property. If we want these tests to run against T9s, then the logic for reading from T9s needs to be altered to be just `response.data`

Reply:
Fixed this and also issue with the tenantId